community/modules/compute/schedmd-slurm-gcp-v6-nodeset/outputs.tf

```diff
@@ -32,6 +32,11 @@ output "nodeset" {
     EOD
   }
 
+  precondition {
+    condition     = var.accelerator_topology == null || var.enable_placement
+    error_message = "accelerator_topology requires enable_placement to be set to true."
+  }
+
```

Comment:
nit. it would be nice to add validation here that `accelerator_topology` if  set, should be in a right format:
Can be parsed, last component is `int` and divides number of gpus. WDYT?